rmarkdown.Rmd

```diff
@@ -167,10 +282,18 @@ like: [@Hoejsgaard2006a]... the reference is inserted onto the bottom of the
 document. You need to add a line like this `bibliography: bibfile.bib` to the top
 YAML header (e.g. right below where you put the `title` and `author`)
 
-## Other resources
+#### Exercise 4: quick R calculations and inserting citations to your document
+Time: Until end of session
+
+Now you are asked:
+
+1. To calculate the correlation between Sepal.Length and Sepal.Width inline (outside of the R chunck). 
```

Comment:
We'll have to show them how to do a correlation. Or give them example code to fill in.

Reply:
yeah I forgot that. But i will say: have a look at the function cor() in the following way: ?cor()